specification/containerregistry/resource-manager/readme.go.md

```diff
@@ -25,6 +26,15 @@ batch:
   - tag: package-2016-06-preview
 ```
 
+### Tag: package-2019-12-preview and go
+
+These settings apply only when `--tag=package-2019-12-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-12-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2019-12-01-preview/$(namespace)
```

Comment:
Please change the output-folder to 
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-12-01-preview/$(namespace)`